Gems/Atom/Component/DebugCamera/Code/Source/CameraComponent.cpp

```diff
@@ -73,11 +73,47 @@ namespace AZ
             AZ::Name viewName = GetEntity() ?
                 AZ::Name(AZStd::string::format("Camera View (entity: \"%s\")", GetEntity()->GetName().c_str())) :
                 AZ::Name("Camera view (unknown entity)");
-            m_view = RPI::View::CreateView(viewName, RPI::View::UsageCamera);
+            
+            uint32_t defaultViewIndex = static_cast<uint32_t>(RPI::ViewType::Default);
+            if (defaultViewIndex < m_cameraViews.size())
+            {
+                m_cameraViews[defaultViewIndex] = RPI::View::CreateView(viewName, RPI::View::UsageCamera);
+            }
+            else
+            {
+                m_cameraViews.insert(m_cameraViews.begin() + defaultViewIndex, RPI::View::CreateView(viewName, RPI::View::UsageCamera));
+            }
+
+            // Add XR pipelines if it is active
+            m_xrSystem = RPI::RPISystemInterface::Get()->GetXRSystem();
+            if (m_xrSystem)
+            {
+                m_numXrViews = m_xrSystem->GetNumViews();
+                AZ_Assert(m_numXrViews <= AZ::RPI::XRMaxNumViews, "Atom only supports two XR views");
+                for (AZ::u32 i = 0; i < m_numXrViews; i++)
+                {
+                    uint32_t xrViewIndex =
+                        i == 0 ? static_cast<uint32_t>(RPI::ViewType::XrLeft) : static_cast<uint32_t>(RPI::ViewType::XrRight);
+                    if (xrViewIndex < m_cameraViews.size())
+                    {
+                        m_cameraViews[xrViewIndex] = RPI::View::CreateView(viewName, RPI::View::UsageCamera | RPI::View::UsageXR);
+                    }
+                    else
+                    {
+                        m_cameraViews.insert(m_cameraViews.begin() + xrViewIndex, RPI::View::CreateView(viewName, RPI::View::UsageCamera| RPI::View::UsageXR));
+                    }
+                }
+            }
```

Comment:
For the future, I wonder if it'd be a more suitable approach to create a "StereoscopicCameraComponent", and somewhere else will use that component if the xr system is available, rather than handle all together in the same class. Thoughts?